source/jormungandr/jormungandr/interfaces/v1/serializer/journey.py

```diff
@@ -92,6 +92,9 @@ class TicketSerializer(PbNestedSerializer):
     found = jsonschema.BoolField()
     cost = CostSerializer(display_none=True)
     links = jsonschema.MethodField(schema_type=LinkSchema(many=True))
+    source_id = jsonschema.Field(
+        schema_type=str, display_none=True, description='Product identifier of the object'
```

Comment:
```suggestion
        schema_type=str, display_none=True, description='Product identifier of the ticket'
```

Reply:
OK. Do I also change the other descriptions in TicketSerializer that say 'object' ?

Reply:
you can yes

Reply:
Done... Except the part after the edit...